app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -69,26 +72,26 @@
 
 .gem-c-image-card__context,
 .gem-c-image-card__metadata {
-  @include core-14;
-  margin: 0 0 $gutter-one-quarter;
-  color: $grey-1;
+  @include govuk-font(14);
+  margin: 0 0 (govuk-spacing(3) / 2);
```

Comment:
At some point (not now) we should have a discussion about the weirder spacing required on GOV.UK, and whether it either needs to be brought in line with the spacing scale or just done as a number - things like this are a bit hard to read sometimes.

Reply:
Agreed